apps/condo/domains/organization/utils/name.utils.js

```diff
@@ -0,0 +1,8 @@
+const SPECIAL_CHARS_REGEXP = /[`!@#$%^&*()_+\-=[\]{};':"\\|,.<>/?~]/
+const NUMBERS_REGEXP = /[0123456789]/
+
+const isValidName = (name) => ![...(name.trim())].every(char => SPECIAL_CHARS_REGEXP.test(char) || NUMBERS_REGEXP.test(char))
```

Comment:
Too complex to understand. Why splitting a chars and what is name is null or undefined?

And most importantly, why not something like `\w{3,}`

Reply:
refactored